nimbus.fml.yaml

```diff
@@ -28,7 +28,7 @@ features:
     defaults: null
   nimbus-validation:
     description: "A feature that does not correspond to an application feature suitable for showing
-    that Nimbus is working."
+      that Nimbus is working."
```

Comment:
@jhugman This was run through a formatter. I have no feelings about the indentation either way, but do the consumers of the fml care?